### DISCUSSION TITLE
?

### DISCUSSION THREAD

**Discussion by gaga675** [bhizusryaaajyaqiu/MindustryExpansion#64] [General]
o mod esta faltando 
•sprits das máquinas {produção}
•como fabricar as unidades {nao tem como fábricar}

**Comment by sryaarf** [comment]
Sorry for The Error At this mod, because this mod is At **Test Pre Alpha** or **Under Development**

> [!WARNING]
> Some Units, Item, and Production For Temporary Can not be use in Champaign mode, because our team still working on the New camping Style

**Reply by gaga675** [reply]
nao se preucupa eu pensei que eu tava bugado, mas parabéns o mod é incrível
